packages/plugins/typescript/resolvers/src/index.ts

```diff
@@ -105,15 +107,23 @@ export type StitchingResolver<TResult, TParent, TContext, TArgs> = {
   }
 
   if (noSchemaStitching) {
-    // Resolver = ResolverFn;
-    defsToInclude.push(`${resolverType} ${resolverFnUsage};`);
+    // Resolver =
+    // | TResult
+    // | Promise<TResult>
+    // | ResolverFn;
+    resolverDefParts.push(`  | ${resolverFnUsage};`);
```

Comment:
Why adding ` | ` to every string? You can add it without it, and then use `.join(' | ')`

Reply:
That's a good point. Now that there are always multiple, it shouldn't be necessary anymore. I'll update the PR

Reply:
Thanks!
@kamilkisiela is the resolvers expert, he'll review it soon :)